project/source/Scripts/Objects/ElectrolysisSetup.gd

```diff
@@ -104,6 +108,8 @@ func slot_filled(slot, object):
 		slot_emptied(slot, object)
 
 func slot_emptied(slot, object):
+	if "Fillable" in object:
```

Comment:
It looks to me like this change is intended to reset the Gel Mold to be fillable again after being removed from the slot.

If that's true, I think we should, instead of using `in object`, check if the object `is_in_group('Gel Boat')` (like you do above in `slot_filled`). Because by checking it this way, we don't actually know for sure whether it is a gel boat, or whether it's some other mystery object that also has a `Fillable` property, which it could theoretically use in a different way and expect not to be modified like this. And if nothing else I also think this would make the code a little more readable.